components/service/glean/README.md

```diff
@@ -61,7 +61,8 @@ class SampleApplication : Application() {
 }
 ```
 
-Once initialized, glean will automatically start collecting and sending its [core metrics](metrics.yaml).
+Once initialized, if collection is enabled, glean will automatically start collecting [core metrics](metrics.yaml)
```

Comment:
Shouldn't we say "baseline metrics" instead of "core metrics" here? (I'm always a little unclear on this).

Reply:
Or I guess core == "baseline" + "ping_info"?  What about saying "internally collected metrics"?

Reply:
No, that was my bad, it's "baseline metrics", as written in the spec. I just got confused :)